src/_data/sites/11ty-bulma starter

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@J-Filip I don't seem to be able to edit the PR, but do you mind renaming the file to 11ty-bulma-starter.json?
I think the build pipeline only greps for .json files.

Reply:
created a new PR #1220 (ooks like I deleted the fork so I had to create another one and resubmit the PR)